org.jacoco.core/src/org/jacoco/core/internal/instr/InstrSupport.java

```diff
@@ -227,4 +228,24 @@ public static void push(final MethodVisitor mv, final int value) {
 		}
 	}
 
+	/**
+	 * Creates {@link ClassReader}.
```

Comment:
We should explain the purpose here:

    Creates a {@link ClassReader} instance even when the given class definition is not yet supported by ASM.

Reply:
Done